dbt/include/spark/macros/materializations/seed.sql

```diff
@@ -69,6 +69,7 @@
     {{ clustered_cols(label="clustered by") }}
     {{ location_clause() }}
     {{ comment_clause() }}
+    {{ tblproperties_clause() }}
```

Comment:
Question: why add it here (`seed.sql`) and not in `adapters.sql` directly under `macro spark__create_table_as`?
It seems it would be a more generic change, no?

Reply:
You are right, @martelli. It makes more sense to put it in the `spark__create_table_as`. I assumed it was there, did not read correctly that the change was done in the `seed.sql.

Reply:
@JCZuurmond Ok. Since I had the fix ready, I've completed a parallel pull request. If this one gets updated and landed, I'll discard mine.